recipes-soletta/soletta/soletta_git.bb

```diff
@@ -6,6 +6,7 @@ DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
 DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl libmicrohttpd mosquitto nodejs"
 DEPENDS += " ${@bb.utils.contains('DISTRO_FEATURES','systemd','systemd','',d)}"
+DEPENDS += " ${@bb.utils.contains('DISTRO_FEATURES','x11','gtk+3','',d)}"
```

Comment:
Yes you are right after reading deeper the documentation about PACKAGECONFIG it looks redundant.

But, do you know if ```PACKAGECONFIG``` guarantees that gtk+3 will be built before Soletta ? 

In the do_configure phase, soletta will look for the gtk+3.0.pc file to check if this dependency is satisfied, if not it will be built without the support we want.